app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -24,8 +24,7 @@ def self.raw_connect(connect_params)
     require 'winrm'
 
     connect_params[:operation_timeout] = 1800
-    winrm = WinRM::Connection.new(connect_params)
-    winrm
+    WinRM::Connection.new(connect_params)
   end
```

Comment:
This is why we typically use the pattern `with_provider_connection { |handle| handle.stuff }` which calls connect, yields the handle, and ensures that the connection gets closed.  I think we should follow that pattern.